lambda-deployment-orca/lambda-deployment-orca.gradle

```diff
@@ -31,16 +31,13 @@ dependencies {
   compileOnly ("org.projectlombok:lombok:1.18.12")
   compileOnly (group: 'com.netflix.spinnaker.kork', name: 'kork-artifacts', version: "${korkVersion}")
   compileOnly (group: 'com.netflix.spinnaker.kork', name: 'kork-plugins-spring-api', version: "${korkVersion}")
-  implementation("com.amazonaws:aws-java-sdk:1.11.877")
-  implementation("org.apache.commons:commons-lang3:3.8.1")
-  implementation("com.github.ben-manes.caffeine:guava:2.8.5")
-  implementation("com.netflix.awsobjectmapper:awsobjectmapper:1.11.877")
-  implementation("org.apache.httpcomponents:httpclient:4.5.9")
-  implementation("org.apache.httpcomponents:httpcore:4.4.11")
-  implementation("com.squareup.okhttp3:okhttp:4.2.2")
-  implementation("com.squareup.okhttp3:okhttp-apache:3.13.1")
+
+  implementation(enforcedPlatform("com.netflix.spinnaker.kork:kork-bom:$korkVersion"))
+  implementation("com.netflix.awsobjectmapper:awsobjectmapper:1.11.802")
```

Comment:
Yes i laso found it, but its not loading without specifying implementation, not sure why - i left the same version 1.11.802 that kork:v7.59.0 uses

Reply:
Aaah, I may see the struggle. New-ish versions of kork are `io.spinnaker.kork`, not `com.netflix.spinnaker.kork`....and the latest version at the moment is [7.142.0](https://repo.maven.apache.org/maven2/io/spinnaker/kork/kork-bom/7.142.0/).

Reply:
This also resolved specifying awsobjectmapper version
```gradle
   implementation(enforcedPlatform("com.netflix.spinnaker.kork:kork-bom:$korkVersion"))
-  implementation("com.netflix.awsobjectmapper:awsobjectmapper:1.11.802")
+  implementation("com.amazonaws:aws-java-sdk")
```

so it’s  either 
```
implementation("com.amazonaws:aws-java-sdk")
```
or
```
implementation("com.netflix.awsobjectmapper:awsobjectmapper")
```

Reply:
The general principle is to be explicit about dependencies...so if the code uses awsobjectmapper, then specify that, even if something else is bringing it in transitively.

Reply:
> I can try 😐 would there be any problems trying to move to the newest kork version?

No idea :). That's the bulk of the effort here.  It may be pretty mechanical.

Reply:
ok